### PR TITLE
[quant][graphmode] Add quantized::conv1d to graphmode

### PR DESCRIPTION
**Pull request by supriyar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #38449 [quant] Support for functional quantized::conv1d
* #38441 [quant][graphmode] Add support for quantized conv1d + relu fusion
* #38438 [quant] Fusion support for conv1d + ReLU
* #38407 [quant] Return default qconfig when backend is 'none'
* **#38341 [quant][graphmode] Add quantized::conv1d to graphmode**
* #38332 [quant] Add quantized::conv1d op benchmarck
* #38283 [quant] Add support for Quantized Conv1d and ConvRELU1d
* #38248 [quant] Add support for quantized::conv1d operator
* #38247 [quant] Implement unsqueeze/squeeze for per-channel qtensor

Summary:

Test Plan:
python test/test_quantization.py TestQuantizeScriptPTSQOps.test_quantized_conv1d

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21554256](https://our.internmc.facebook.com/intern/diff/D21554256)

### CODE REVIEW

File: test/quantization/test_quantize_script.py [hunk 1]
```diff
@@ -1100,7 +1101,10 @@ class TestQuantizeScriptPTSQOps(JitTestCase):
     for individual ops end to end.
     """
     def _test_op_impl(self, module, data, quantized_op):
-        qconfig_dict = {'': get_default_qconfig('fbgemm')}
+        qengine = torch.backends.quantized.engine
+        if qengine == 'none':
+            qengine = 'fbgemm'
```
Comment (jerryzh168) [inline test/quantization/test_quantize_script.py:1106]: why not pass in qengine here?

Comment (supriyar) [reply test/quantization/test_quantize_script.py:1106]: We would still have to check for `none` case for OS like macos where neither is enabled to get the correct qconfig.

Comment (jerryzh168) [reply test/quantization/test_quantize_script.py]: can we just add this to get_default_qconfig?

Comment (supriyar) [reply test/quantization/test_quantize_script.py]: yeah I was planning to do it in a follow up PR

Comment (supriyar) [reply test/quantization/test_quantize_script.py]: See #38407

**Review by jerryzh168 — APPROVED** [review]

**Pull request closed without merging**